scootapi/client.go

```diff
@@ -0,0 +1,101 @@
+package scootapi
```

Comment:
its similar but doesn't expose Dial() just the CloudScoot Interface, but I suspect after taking a quick look workerapi/client will have the same problem that this client had before.  If the connection dies due to broken pipe or an unrecoverable error, the connection needs to be closed and re-established.  Otherwise every subsequent call will fail.